apps/condo/domains/user/components/UserMenu.tsx

```diff
@@ -37,8 +39,8 @@ export const UserMenu: React.FC = () => {
 
     const handleSignOutClick = useCallback(async () => {
         await auth.signOut()
-        await router.push('/auth/signin')
-    }, [auth])
+        await router.push(goToAfterLogout)
```

Comment:
It's better to extract it to a function. We can add some logic with the `next` redirect url. When you go to some page `/ticket/123` -> `/auth?next=/ticket/123`